dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -41,6 +41,7 @@ To set up Prebid to serve ads into your AMP pages, you'll need:
 
 + [AMP content page](#amp-content-page): This is where your content lives.
 + [HTML Creative](#html-creative): This is the creative your Ad Ops team puts in your ad server.
++ [User Sync in AMP](#user-sync-in-amp): This is what you need to put on page to do User Sync.
```

Comment:
"This is the `amp-iframe` pixel that must be added to your AMP page to sync users with Prebid Server."

Reply:
Updated